CHANGELOG.md

```diff
@@ -3,7 +3,10 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
-### 4.12.3.0.1
+### 4.12.4.0.0
+- This version of the adapter has been certified with AppLovin SDK 12.4.0.
+
+### 4.12.3.1.0
```

Comment:
_Sigh_. Previous adapter had the version typed out in here incorrectly by me, so I fixed that here too.